.github/pull_request_template.md

```diff
@@ -0,0 +1,23 @@
+## Problem
+
+- Describe the problem this PR solves here.
+- If you're fixing an issue, say "Fixes #X".
+
+## Solution
+
+- Describe the solution used to overcome the problem.
+
+## Additional Information
+
+- Any additional information needed to better understand this PR.
+
+## Breaking Changes (if applicable)
+
+- What changes were introduced that would break backwards compatibility?
+- How would a user modify their code to adjust for these breaking changes?
```

Comment:
Just added this section as well. 
Thoughts?

Reply:
I'm a bit nervous about this; I can see the value but this is a lot of friction and complexity to impose.

Reply:
Yea, this is fair.
It probably just adds noise that most PRs don't even have to worry about.